hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/ZookeeperCounterReporter.java

```diff
@@ -53,6 +54,28 @@ public void report(SortedMap<String, Gauge> gauges,
         for (Map.Entry<String, Counter> entry : counters.entrySet()) {
             reportCounter(entry.getKey(), entry.getValue());
         }
+
+        meters
```

Comment:
For consistency, I suggest to write this forEach in the same way as loop above.

Reply:
After your comment I decided to simplify this first loop.